atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -157,7 +157,7 @@ def run(self):
 
         annotations = {
             "dockerfile": open(self.workflow.builder.df_path).read(),
-            "artefacts": self.get_pre_result(DistgitFetchArtefactsPlugin.key),
+            "artefacts": self.get_pre_result(DistgitFetchArtefactsPlugin.key) or [],
```

Comment:
We shouldn't store this annotation at all any more. fetch_artifacts no longer returns a value, and osbs-client doesn't care about this annotation.